IlmBase/config/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ include(CheckIncludeFiles)
 include(CheckSymbolExists)
 include(CheckLibraryExists)
 include(CheckStructHasMember)
+include(../../cmake/JoinPaths.cmake)
```

Comment:
My preference would be that the cmake path be incorporated into the cmake include path from the root path, rather than using a relative path like this. Is there a reason not to do that?

Reply:
Do you mean something like `set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake")`?

Reply:
yes, exactly. at a higher more global level than IlmBase/config.

Reply:
Hi @jtojnar, as we are currently on the verge of reorganizing this repo due to the release of Imath as a separate library, I think it is reasonable to deal with cmake paths and so on after that occurs.